### PR TITLE
build(docs-infra): update to latest CLI beta

### PR DESCRIPTION
**Pull request by filipesilva**
Notable feature include:
- Web Worker support (https://github.com/angular/angular-cli/pull/13700)
- Seamless NGCC support (https://github.com/angular/angular-cli/pull/13968)
- Sass (dart-sass) support (https://github.com/angular/angular-cli/pull/13950)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by filipesilva** [comment]
This PR as is currently errors out:
```
kamik@RED-X1C6 MINGW64 /d/work/angular/aio (cli-webworker)
$ yarn ~~build
yarn run v1.12.3
$ ng build --configuration=stable
Cannot read property 'endsWith' of undefined
TypeError: Cannot read property 'endsWith' of undefined
    at NgBuildAnalyticsPlugin._collectBundleStats (D:\work\angular\aio\node_modules\@angular-devkit\build-angular\plugins\webpack\analytics.js:174:32)
    at NgBuildAnalyticsPlugin._done (D:\work\angular\aio\node_modules\@angular-devkit\build-angular\plugins\webpack\analytics.js:211:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\work\angular\aio\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:18:1)
    at AsyncSeriesHook.lazyCompileHook (D:\work\angular\aio\node_modules\tapable\lib\Hook.js:154:20)
    at onCompiled (D:\work\angular\aio\node_modules\webpack\lib\Compiler.js:234:21)
    at hooks.afterCompile.callAsync.err (D:\work\angular\aio\node_modules\webpack\lib\Compiler.js:630:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\work\angular\aio\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\work\angular\aio\node_modules\tapable\lib\Hook.js:154:20)
    at compilation.seal.err (D:\work\angular\aio\node_modules\webpack\lib\Compiler.js:627:30)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\work\angular\aio\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:21:1)
    at AsyncSeriesHook.lazyCompileHook (D:\work\angular\aio\node_modules\tapable\lib\Hook.js:154:20)
    at hooks.optimizeAssets.callAsync.err (D:\work\angular\aio\node_modules\webpack\lib\Compilation.js:1325:35)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\work\angular\aio\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:10:1)
    at AsyncSeriesHook.lazyCompileHook (D:\work\angular\aio\node_modules\tapable\lib\Hook.js:154:20)
    at hooks.optimizeChunkAssets.callAsync.err (D:\work\angular\aio\node_modules\webpack\lib\Compilation.js:1316:32)
    at _err2 (eval at create (D:\work\angular\aio\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:37:1)
```

This will be fixed by https://github.com/angular/angular-cli/pull/14104. Until then, for local development, the code changes in that PR can be added to `node_modules\@angular-devkit\build-angular\plugins\webpack\analytics.js`.

Then the build errors with:
```
kamik@RED-X1C6 MINGW64 /d/work/angular/aio (cli-webworker)
$ yarn ~~build
yarn run v1.12.3
$ ng build --configuration=stable

Date: 2019-04-08T10:16:22.066Z
Hash: 9b6dd3de5688c7463f48
Time: 9358ms
chunk {0} runtime.741402d1d47331ce975c.js, runtime.741402d1d47331ce975c.js.map (runtime) 1.46 kB [entry] [rendered]
chunk {1} main.4af9b61479361f268d39.js, main.4af9b61479361f268d39.js.map (main) 182 bytes [initial] [rendered]
chunk {2} polyfills.961ffc045f11208a7197.js, polyfills.961ffc045f11208a7197.js.map (polyfills) 189 bytes [initial] [rendered]
chunk {3}  (styles) [initial] [rendered]

ERROR in ./src/styles.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):

undefined
   ^
      Parent "table tbody > tr >" is incompatible with this selector.
   ╷
73 │     &:last-child td {
   │     ^^^^^^^^^^^^^^^^
   ╵
  src\styles\2-modules\_table.scss 73:5         @import
  src\styles\2-modules\_modules-dir.scss 23:12  @import
  src\styles\main.scss 14:9                     @import
  stdin 3:9                                     root stylesheet
      in D:\work\angular\aio\src\styles\2-modules\_table.scss (line 73, column 5)
    at runLoaders (D:\work\angular\aio\node_modules\webpack\lib\NormalModule.js:301:20)
    at D:\work\angular\aio\node_modules\loader-runner\lib\LoaderRunner.js:367:11
    at D:\work\angular\aio\node_modules\loader-runner\lib\LoaderRunner.js:233:18
    at context.callback (D:\work\angular\aio\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
    at render (D:\work\angular\aio\node_modules\sass-loader\lib\loader.js:52:13)
    at Function.$2 (D:\work\angular\aio\node_modules\sass\sass.dart.js:24366:48)
    at wN.$2 (D:\work\angular\aio\node_modules\sass\sass.dart.js:15294:15)
    at uS.vr (D:\work\angular\aio\node_modules\sass\sass.dart.js:9034:42)
    at uS.vq (D:\work\angular\aio\node_modules\sass\sass.dart.js:9036:32)
    at iy.uD (D:\work\angular\aio\node_modules\sass\sass.dart.js:8384:46)
    at ur.$0 (D:\work\angular\aio\node_modules\sass\sass.dart.js:8526:7)
    at Object.eG (D:\work\angular\aio\node_modules\sass\sass.dart.js:1512:80)
    at ad.ba (D:\work\angular\aio\node_modules\sass\sass.dart.js:8447:3)
    at iM.ba (D:\work\angular\aio\node_modules\sass\sass.dart.js:8377:25)
    at iM.cv (D:\work\angular\aio\node_modules\sass\sass.dart.js:8364:6)
    at px.cv (D:\work\angular\aio\node_modules\sass\sass.dart.js:8154:35)
ERROR in src/app/search/search-worker.ts(76,5): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'PageInfo' has no compatible call signatures.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

There are two errors here:
- A sass error, where dart-sass behaviour differs from the old node-sass (more info at https://github.com/sass/dart-sass#behavioral-differences-from-ruby-sass). The old node-sass processor can be set by following the instructions in https://github.com/angular/angular-cli/pull/13950, but it would be better to adhere to dart-sass instead since it is stricter.
- A TS typings error in `search-worker.ts`, which is still in the process of being converted to TS.

**Comment by filipesilva** [comment]
@clydin can you provide guidance on how to enable analytics for AIO locally and in CI?

**Comment by gkalpak** [comment]
@filipesilva: FYI, I've fixed the Scss issues in #28530 (if you want to copy it over): https://github.com/angular/angular/commit/b4f65da90

**Comment by filipesilva** [comment]
Note: both @petebacondarwin and @gkalpak worked on this PR, but at some point I had to `git reset` and remake the commits so their contributions were lost in mine.

**Review by gkalpak — APPROVED** [review]
:tada:

File: aio/src/app/search/search-worker.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import { WebWorkerMessage } from '../shared/web-worker-message';
+import * as lunr from 'lunr';
+
+// TODO: remove definition for importScripts, it's not used anymore.
```
Comment (gkalpak) [inline aio/src/app/search/search-worker.ts]: OOC, what does this comment refer to? What definitions? Remove from where?

Comment (filipesilva) [reply aio/src/app/search/search-worker.ts]: This comment shouldn't be here anymore. It referred to things in the old JS version:
```
/* global importScripts, lunr */

// NOTE: This needs to be kept in sync with `ngsw-config.json`.	
importScripts('/generated/lunr.min.js');
```

Will track those down and remove them.

Comment (filipesilva) [reply aio/src/app/search/search-worker.ts]: Huh, it's a native function that's present in workers: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts. Didn't know that. Will just remove the comment then.

**Pull request closed without merging**